lib/fast_stats.rb

```diff
@@ -1,5 +1,7 @@
+require 'active_support/all'
 require "fast_stats/version"
 
 module FastStats
-  # Your code goes here...
+  require 'fast_stats/mean'
+  require 'fast_stats/means'
```

Comment:
These require statements would typically be at the top of the file as opposed to inside the `module`. I don't think there's any problem with this; it's just a bit odd.

Reply:
Oh, this is the way we structure the FB gem, why did we do that?

Reply:
Oh huh, I don't know. I'm guessing I just didn't comment on it before.